students/youjin/westagram/users/urls.py

```diff
@@ -1,7 +1,8 @@
 from django.urls import path
 
-from users.views import UserView
+from users.views import UserView, LoginView
 
 urlpatterns = [
     path('', UserView.as_view()),
+    path('/login', LoginView.as_view()),
```

Comment:
`suggestion`과 같이 해당 `View`가 어떤 기능을 하는지 네이밍 또한 정확히, 엔드포인트 주소 또한 한 눈에 알아보기 쉽게 작성해주시면 좋을것 같아요~ `views.py`도 이와 같이 수정해주면 좋을 것 같습니다!
```suggestion
    path('/signup', SignUpView.as_view()),
    path('/login', LoginView.as_view()),
```

Reply:
로그인뿐만 아니고, 회원가입 기능 관한 네이밍도 수정했습니다! 감사합니다~

Reply:
네 맞습니다! 하나의 View안에 post부터 get까지 정말 다양한 기능을 넣을 수 있겟지만, 그럴 경우 View 이름을 좀 더 포괄적인 단어로 표현하시면 되겠습니다!